test/helpers/fixtures.js

```diff
@@ -0,0 +1,22 @@
+import db from '../../src/services/db';
+import * as baseSql from '../../src/base/sql';
+
+const data = {
+  state: require('../fixtures/state')
+};
+
+export default function fixtures () {
+  // Array of promises representing table-insert batch transactions
+  const insertPromises = [];
+  for (const table in data) {
+    // Insert data for each table
+    insertPromises.push(db.tx(t => {
+      return t.batch(data[table].map(insert => {
+        const fields = Object.keys(insert);
```

Comment:
Ah boo. Okay. I'd throw the `pgp` export in `src/base/pgp.js` and import it in `services/db`.

`src/services/` is meant to hold files related to connections with external services. It doesn't make sense (to my mind) to export the constructor for connecting to one from there. However, `src/base` seems like the perfect location.